.gitignore

```diff
@@ -0,0 +1 @@
+bikeshed/
```

Comment:
Why?

Reply:
This is much more for my personal benefit cuz I do keep bikeshed in a repo in the code. I will go ahead and take this out if it causes controversy.